Source/Extensions/Blazorise.SignaturePad/wwwroot/signaturepad.js

```diff
@@ -0,0 +1,153 @@
+import { getRequiredElement } from "../Blazorise/utilities.js?v=1.2.3.0";
+import "./vendors/sigpad.js?v=1.2.3.0";
+
+const _instances = [];
+
+export function initialize(dotNetAdapter, element, elementId, options) {
+    element = getRequiredElement(element, elementId);
+
+    if (!element)
+        return;
+
+    const sigpad = new SignaturePad(element, {
+        minWidth: options.minLineWidth || 0.5,
+        maxWidth: options.maxLineWidth || 2.5,
+        backgroundColor: options.backgroundColor || "rgba(0,0,0,0)",
+        penColor: options.penColor || "black",
+        velocityFilterWeight: options.velocityFilterWeight || 0.7,
+        dotSize: options.dotSize,
+        throttle: options.throttle || 16,
+        minPointDistance: options.minPointDistance || 0.5,
+    });
+
+    if (options.value) {
+        sigpad.fromData(options.value, { clear: false });
+    }
+
+    if (options.readOnly === true) {
```

Comment:
no option to start as readonly?

Reply:
This is the option to as readonly.

Reply:
Unfortunately, there is no such API in SignaturePad. So we need to first create an instance and then disable it it requested.

Reply:
Well ok then. :)